subsys/net/lib/http/http_client.c

```diff
@@ -149,6 +149,7 @@ static int on_url(struct http_parser *parser, const char *at, size_t length)
 	struct http_request *req = CONTAINER_OF(parser,
 						struct http_request,
 						internal.parser);
+
```

Comment:
This seems to be unrelated change not related to this commit. Please remove it.

Reply:
it is modified by uncrustify, I will move it to a new commit, maybe remove it if uncrustify has no complaints. :p 
https://docs.zephyrproject.org/latest/contribute/index.html#uncrustify

Reply:
introduced 51dbeb6 to fix coding style issues, the new line is needed.